tests/system/action/meeting/test_create.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict
+
+from tests.system.action.base import BaseActionTestCase
+
+
+class MeetingCreateActionTest(BaseActionTestCase):
+    def helper(self, datapart: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Please give the function a more descriptive name

Reply:
Renamed in `basic_test`